# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1247166274 14400
# Node ID cd95695a0a89f9724b285c124253382ccfbe83ea
# Parent  4290384284b7d01e78cea98cffc4dcbc6ce2bc76
configure.in: don't use system strftime on MinGW systems

diff -r 4290384284b7 -r cd95695a0a89 ChangeLog
--- a/ChangeLog	Wed Jul 08 14:06:53 2009 -0400
+++ b/ChangeLog	Thu Jul 09 15:04:34 2009 -0400
@@ -1,3 +1,7 @@
+2009-07-09  John W. Eaton  <jwe@octave.org>
+
+	* configure.in: Don't use system strftime on MinGW systems.
+
 	2009-07-21  Jaroslav Hajek  <highegg@gmail.com>
 
 	Version 3.2.2 released.
diff -r 4290384284b7 -r cd95695a0a89 configure.in
--- a/configure.in	Wed Jul 08 14:06:53 2009 -0400
+++ b/configure.in	Thu Jul 09 15:04:34 2009 -0400
@@ -1640,8 +1640,8 @@
 esac
 
 case "$canonical_host_type" in
-  *-*-msdosmsvc)
-    ## The %T format specifier for strftime is reportedly broken,
+  *-*-msdosmsvc | *-*-mingw*)
+    ## The %T and %e format specifiers for strftime are not implemented
     ## so use our version.  We could use an actual configure test
     ## for this.
   ;;
